dataset/dictionary/story.json5

```
@@ -323,55 +323,62 @@
   {
     en: "Darshan",
     ja: "学派 / 学院",
+    zhCN: "学派 / 学院",
     tags: [ "sumeru" ],
     notes: "英語版では学派も学院も区別されずに Darshan と呼ばれているようである。原義はサンスクリット語で (特に聖人や神像などを) 見ること",
   },
   {
     en: "six Darshans",
     ja: "六大学派",
+    zhCN: "六大学派",
     tags: [ "sumeru" ],
   },
   {
     en: "Amurta",
     ja: "生論派 / アムリタ学院",
-    // zhCN: "生论派", // TODO zh-translation for アムリタ学院
+    zhCN: "生论派 / 阿弥利多学院",
     tags: [ "sumeru" ],
     notes: "英語版では「生論派」と「アムリタ学院」は区別されていないようである。\"Amurta Darshan\" と Darshan (学派 / 学院) を付けて表記される場合もある",
   },
   {
     en: "Rtawahist",
     ja: "明論派 / ルタワヒスト学院",
+    zhCN: "明论派 / 梨多梵谛学院",
     tags: [ "sumeru" ],
     notes: "英語版では「明論派」と「ルタワヒスト学院」は区別されていないようである。\"Rtawahist Darshan\" と Darshan (学派 / 学院) を付けて表記される場合もある",
   },
   {
     en: "Spantamad",
     ja: "素論派 / スパンタマッド学院",
+    zhCN: "素论派 / 悉般多摩学院",
     tags: [ "sumeru" ],
     notes: "英語版では「素論派」と「スパンタマッド学院」は区別されていないようである。\"Spantamad Darshan\" と Darshan (学派 / 学院) を付けて表記される場合もあると思われる",
   },
   {
     en: "Driyosh",
     ja: "ダリオッシュ",
+    zhCN: "游学者", // TODO: Need check
     tags: [ "sumeru", "title" ],
     notes: "遊学者を意味する言葉。スメール教令院の素論派に多いとされる",
   },
   {
     en: "Haravatat",
     ja: "知論派 / ハルヴァタット学院",
-    // zhCN: "知论派", // TODO zh-translation for ハルヴァタット学院
+    zhCN: "知论派 / 室罗婆耽学院", //
```

Can you delete the comment?

```suggestion
    zhCN: "知论派 / 室罗婆耽学院",
```